### PR TITLE
Features/homepage

### PR DESCRIPTION
**Pull request by karvel-code**
The pull request is on a NewYork Times webpage clone developed by nganifaith and karvel-code. The webpage contains the following features:
-Header.
-Main content.
-Footer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,524 @@
...
+        <nav>
+        <div class = 'header-left'>
+            <i class="fa fa-bars" aria-hidden="true"></i>
+            <i class="fa fa-search" aria-hidden="true"></i>
+        
+            <a href=""><b>SPACE & COSMOS</b></a>
+        </div>
+        <div class='header-center'>
+            <img src= "Assets/images/NewYorkTimes.png" alt="New york Times">
+        </div>
```
Comment (jubaan) [inline index.html]: - [ ] TheNYT logo should be a link also

File: index.html [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+        <div class = 'header-right'>
+            <button>Subcribe Now</button>
+            <button>Login</button>
```
Comment (jubaan) [inline index.html]: - [ ] This should be links also

File: index.html [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+                        <p>The Learning Network</p>
+                        <p>Tools & Services</p>
+                        <p>Multimedia</p>
+                        <p>Photography</p>
+                        <p>Video</p>
+                        <p>Newletters</p>
+                        <p>TimesMachine</p>
+                        <p>NYT Store</p>
+                        <p>Times Journeys</p>
+                        <p>Manage My Account</p>
```
Comment (jubaan) [inline index.html]: - [ ] This elemenst are part of a menu, so they should be links

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @karvel-code and @nganifaith. Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [ ] The PR message should be descriptive and meaningful, please *edit* the actual. Recommendations give a brief summary of the things you have done, the features, and/or the goals of the project
- [ ] Social media icons out of place
image
- [ ] Image broken and both image and articles should be links
image

- [ ] Images should have the with of the paragraph, and the paragraphs should be all aligned
image

- [ ] Images and article titles should be links, the sections don't look like the original design, the images should be horizontally aligned.
image


### [OPTIONAL]  :yellow_circle:  

- [ ] The readme has most of the relevant information and that's great!, but it is difficult to read in the *Getting started* section, it contains a number of steps but the way is presented makes it hard to follow. To fix this instead of having a paragraph with all the information please put each step in a new line, so is visually more appealing and readable.
- [ ] The Author section has some markdown syntax issues, and change the keyword `githubhandle` for your actual github account name. example:
  *BAD*
  Julio: [@githubhandle] (https://www.github.com/jubaan) // this has a space between `]` and `(`
  Julio: [@githubhandle](https://www.github.com/jubaan) // this uses a generic username

  *GREAT*
  Julio: [@jubaan](https://www.github.com/jubaan) // it uses my name and is formatted this way `[@jubaan](https://www.github.com/jubaan)` personal username and no spaces between square brackets and paretheses.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,546 @@
...
+
+    <main>
+        <div class = "content-head">
+            <h1>Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+            <div class = "share">
+                <i class="fa fa-facebook social-share" aria-hidden="true"></i>
+                <i class="fa fa-twitter social-share" aria-hidden="true"></i>
+                <i class="fa fa-envelope social-share" aria-hidden="true"></i>
+                <i class="fa fa-share social-share" aria-hidden="true"></i>
+                <i class="fa fa-bookmark-o book-mark" aria-hidden="true"></i>
```
Comment (mcrd25) [inline index.html:40]: [OPTIONAL]
These should be links

File: index.html [hunk 5]
```diff
@@ -0,0 +1,546 @@
...
+                </div>
+                <div>
+                    <a href="#"><img src = "./Assets/images/side-img2.jpg" alt = "A Medical personal"></a>
+                    <a href="#">Why I Got the Russian Vaccine</a>
+                </div>
+                <div>
+                    <a href="#"><img src = "./Assets/images/side-img3.jpg" alt = "Baseball Coach with a bat"></a>
+                    <a href="#">Pandemic Victories for College Sports Have All Come With Caveats</a>
+                </div>
+            </aside>
```
Comment (mcrd25) [inline index.html:99]: [OPTIONAL]
The sizing for this is slightly bigger than the original

**Review by mcrd25 — APPROVED** [review]
# Changes Approved :heavy_check_mark: 

Hi Team! :wave: 

You've done enough to warrant a :+1: . Check out my additional comments though! :slightly_smiling_face: 


***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

If you liked my review, consider upskilling me on LinkedIn. And please do connect. [My LinkedIn](https://www.linkedin.com/in/mayadouglas/)


*Happy coding 😄,*
**[--Maya Douglas](mayacdouglas.com)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)
